src/plugins/data_source_management/public/components/utils.ts

```diff
@@ -120,7 +157,8 @@ export const fetchDataSourceConnections = async (
   http: HttpSetup | undefined,
   notifications: NotificationsStart | undefined,
   directQueryTable: boolean,
-  hideLocalCluster: boolean = false
+  hideLocalCluster: boolean,
```

Comment:
i think the node should be complaining at you in your logs, i had to add this default value because at some places some people didn't pass this param.